algorithms/src/snark/varuna/ahp/indexer/indexer.rs

```diff
@@ -140,15 +140,15 @@ impl<F: PrimeField, SM: SNARKMode> AHPForR1CS<F, SM> {
 
         crate::snark::varuna::ahp::matrices::pad_input_for_indexer_and_prover(&mut ics)?;
 
-        let a = ics.a_matrix()?;
-        let b = ics.b_matrix()?;
-        let c = ics.c_matrix()?;
+        let IndexerConstraintSystem { a, b, c, num_public_variables, num_private_variables, num_constraints } = ics;
+
+        let a = into_matrix_helper(a, num_public_variables)?;
+        let b = into_matrix_helper(b, num_public_variables)?;
+        let c = into_matrix_helper(c, num_public_variables)?;
 
         end_timer!(padding_time);
 
-        let num_padded_public_variables = ics.num_public_variables();
-        let num_private_variables = ics.num_private_variables();
-        let num_constraints = ics.num_constraints();
```

Comment:
Move these lines to inside the `if` statement below, otherwise debug mode will not compile.

Reply:
These were only removed because they are already bound with the same names in [L143](https://github.com/AleoHQ/snarkVM/pull/2240/files#diff-308e0d2507e765d9aa886da9e3edd2a70deef59ef5d794aa10b842f2974e5051R143); I checked in debug mode and it compiles just fine.